config/webpack/terra-dev-site.webpack.config.js

```diff
@@ -86,22 +50,8 @@ const devSiteConfig = (env = {}, argv = {}) => {
   // Generate the files need to spin up the site.
   generateAppConfig(siteConfig, production, verbose);
 
-  // Get the default package name.
-  const packageName = siteConfig.npmPackage.name;
-
   // Is hot reloading enabled?
-  const { hotReloading, monoRepo, webpackAliasOptions } = siteConfig;
-
-  // Add the auto aliases for the current packages and all mono-repo packages.
-  const alias = {
-    ...aliasMonoRepoPackages(monoRepo, hotReloading, webpackAliasOptions, production),
-    ...aliasCurrentPackage(packageName, processPath, hotReloading, webpackAliasOptions, production),
-  };
-
-  if (verbose) {
-    // eslint-disable-next-line no-console
-    console.log('Generated Aliases', alias);
-  }
+  const { hotReloading, webpackAliasOptions } = siteConfig;
```

Comment:
Can we add some documentation on `webpackAliasOptions`?

Reply:
It's not new: https://github.com/cerner/terra-dev-site/blob/master/config/site/site.config.js#L48-L56

But I'm going to delete it.

Reply:
ah didn't realize it came from the site config